lib/registration.pm

```diff
@@ -620,6 +620,7 @@ sub install_docker_when_needed {
             assert_script_run('zypper se docker || zypper -n ar -f http://download.suse.de/ibs/SUSE:/SLE-12:/Update/standard/SUSE:SLE-12:Update.repo');
         }
         else {
+            zypper_call('in SUSEConnect');
```

Comment:
probably better to insert it inside `add_suseconnect_product` so it will fix problem in more generic way ?